website/src/pages/index.tsx

```diff
@@ -305,7 +306,7 @@ function Configure() {
                   Container Engines
                 </h2>
                 <p className="leading-relaxed text-base">
-                  Handle multiple container engines at the same time (Podman, Docker, Lima...).
+                  Handle multiple container engines at the same time: Podman, Docker, Lima.
```

Comment:
```suggestion
                  Handle multiple container engines at the same time: Podman, Docker, and Lima.
```

Small nitpick otherwise LGTM